spec/repository_spec.rb

```diff
@@ -26,3 +26,17 @@
     expect(EXTRA_FILES).to eq([])
   end
 end
+
+describe "The file \"renamed-styles.json\"" do
+  it "must be valid JSON" do
+    expect(JSON_IS_VALID).to be true
+  end
```

Comment:
Well, I get:

* on Windows 7, Ruby 2.1.0, SSD, with https://github.com/adoxa/ansicon installed (for proper color output, still prints thousands of lines):
```
Finished in 2 minutes 44.6 seconds (files took 38.24 seconds to load)
131958 examples, 0 failures
```

* after `SET CONTINUOUS_INTEGRATION=true` (per https://github.com/travis-ci/travis-ci/issues/1337):
```
Finished in 1 minute 12.98 seconds (files took 33.74 seconds to load)
131958 examples, 0 failures
```
(wow! Looks like the standard progress bar slows things down quite a bit; I'll check whether I get any performance improvement locally with `CONTINUOUS_INTEGRATION=true` on OS X later)

* after disabling style validation:
```
Finished in 37.49 seconds (files took 33.07 seconds to load)
123791 examples, 0 failures
```

On my Mac I also usually get something around 1 minute 10 seconds, with validation. So it looks like validation does make a big difference.